libs/components/molecules/src/lib/interactive-svg/interactive-svg.component.ts

```diff
@@ -72,7 +72,7 @@ export interface NodeTooltipData {
 })
 export class InteractiveSvgComponent implements OnDestroy {
   /** SVG url */
-  @Input() url?: string;
+  @Input() url?: string | null;
```

Comment:
Why add null?

Reply:
I was trying something earlier, forgot to remove it. Will get rid of it